### ISSUE TITLE
Shorter commands for the ansible installation pipeline

### ISSUE THREAD

**Issue by wintered** [wintered/dotfiles#6]
The ansible installation pipeline is great, however commands like  

`ansible-playbook -i dotfiles_installation/localhost_inventory.yml dotfiles_installation/install/install.yml
`
are simply too long (even need vertical scrolling bars in the github README). Find a way to make this more compact that still obeys the ansible conventions and is not overly cryptic.

**Comment by MFreidank** [comment]
we could ship a small script that exposes all command line options of `ansible` but uses the call 
outlined above as default. 
What's your thoughts?

**Comment by wintered** [comment]
Yes what you mention is a good idea. How would you call the script? What is the convention? 
Maybe something like:

`
python setup.py <ansible-playbooks-options-and-args>
`

What do you think?

**Comment by MFreidank** [comment]
I think `setup.py` is a somewhat protected name used to distribute pure
python packages - which is not what we do here, so I think we should call
the script something else.

Maybe:
python dotfiles.py install/uninstall [ansible-options]

Which resolves to:
ansible-playbooks -i $INVENTORY_FILE tasks/[install/uninstall].yml [further
ansible options, e. G. tags]

With what I propose a simple: `python dotfiles.py install` would set all
dotfiles up. We could even use the script to also install ansible [with an
option to use an existing executable, to support rare cases with strange
setup] , which would allow users to only have to worry about having python
available.
What do you think about this?

Am Di., 19. Feb. 2019, 20:20 hat Dominik Winterer <notifications@github.com>
geschrieben:

**Comment by wintered** [comment]
> We could even use the script to also install ansible [with an option to use an existing executable, to support rare cases with strange setup] , which would allow users to only have to worry about having python available.

Sounds good.

**Comment by wintered** [comment]
@MFreidank, offline you indicated that you were already working on this issue. Did you mean the recent post (which is the way we should do it) or uncommited changes on your side? If did not mean the latter, I would start coding.

**Comment by MFreidank** [comment]
@wintered Only saw this now, sorry for the delayed reply. I do remember that I have some uncommitted changes, but if you already started coding, feel free to go on, it was not that advanced yet.

**Comment by MRocholl** [comment]
also second a nice script to execute the ansible playbooks. 
Nevertheless this is definitely not a priority for now. 
Refactoring most of the zsh and include ansible roles that actually cover all the binaries that are called by anywhich zsh function comes first.
